butt.spec

```diff
@@ -23,8 +23,8 @@
 #
 
 # Conditional build support
-# add --with aac option, i.e. disable AAC by default
-%bcond_with aac
+# add --without aac option, i.e. enable AAC by default
```

Comment:
What's the reason for this change? I don't quite understand to what build process you refer to with 

> Activate and install fdk-aac per default since the build process doesn't have an easy way to deactivate this.

Reply:
The `./configure` file checks for fdk-aac in any case and fails the build. I didn't find an obvious way to compile butt without aac support. I don't believe that the `--without` flags for other codecs even work (but did do any further testing). I did quite some hacking but coudn't figure out how to do a source build of butt without fdk-aac.

Reply:
Ah ok, thanks for testing then! Actually, the butt build script doesn't have a way to explicitly enable or disable the specific codecs (it lacks any `--enable-` / `--disable-` or `--with-` / `--without-` options), it simply checks if the libraries are available and enables them if found. I wasn't aware that it doesn't work for AAC (I probably had the libs already installed while developing the spec file).